server/server.js

```diff
@@ -1,4 +1,5 @@
 let fs = require('fs')
+const {header, validationResult} = require('express-validator')
```

Comment:
can we use something like `const Validator = require(...)` in order to avoid ambiguous global variables?